src/huggingface_hub/hf_file_system.py

```diff
@@ -377,19 +447,72 @@ def cp_file(self, path1: str, path2: str, revision: Optional[str] = None, **kwar
 
     def modified(self, path: str, **kwargs) -> datetime:
         info = self.info(path, **kwargs)
-        if "last_modified" not in info:
-            raise IsADirectoryError(path)
-        return info["last_modified"]
+        return info["last_commit"]["date"]
 
-    def info(self, path: str, **kwargs) -> Dict[str, Any]:
-        resolved_path = self.resolve_path(path)
+    def info(self, path: str, refresh: bool = False, revision: Optional[str] = None, **kwargs) -> Dict[str, Any]:
```

Comment:
What if we want the info with `detail=False` ?

Reply:
This would make calls to `isdir` pretty slow no ? We use it a lot in `datasets`

Reply:
Ideally `isdir` calls `info()` with `detail=False` and `info()` doesn't need to call `expand=True` in this case, which is slow on the Hub side.

It's a bit out of scope of this PR though, we can see later

Reply:
Made the change to allow calling `.info` with `detail=False` (through the `kwargs`) to support efficient `exists`/`isdir`/`isfile`, but `detail` is not epoxsed as the method parameter to follow the spec.